modules/ROOT/pages/basic-authentication-simple-concept.adoc

```diff
@@ -19,19 +19,15 @@ The policy follows https://tools.ietf.org/html/rfc7617[basic HTTP authentication
 
 When an error is encountered, the policy returns an "WWW-Authenticate" HTTP header field using the following format:`WWW-Authenticate: Basic realm="mule-realm"`.
 
-== Configuring Policy Parameters
+== Creating Requests for APIs With Basic Authentication Policies 
```

Comment:
```suggestion
== Creating Requests for APIs with Basic Authentication Policies 
```
Here and in the previous title, does "with" mean "by using" or "that include"? Consider clarifying.

Reply:
Creating Requests for APIs with Basic Authentication Policies Applied